wls-vorfaelleundvorkommnisse-service/src/main/java/de/muenchen/oss/wahllokalsystem/vorfaelleundvorkommnisseservice/rest/ereignis/EreignisController.java

```diff
@@ -20,69 +20,67 @@
                     ),
-                    @ApiResponse(
-                            responseCode = "500", description = "Probleme bei der Kommunikation mit dem externen System von dem die Daten importiert werden",
-                            content = { @Content(mediaType = "application/json", schema = @Schema(implementation = WlsExceptionDTO.class)) }
-                    ),
                     @ApiResponse(
                             responseCode = "204", description = "Keine Daten vom Fremdsystem geliefert",
                             content = @Content(schema = @Schema())
+                    ),
+                    @ApiResponse(
+                            responseCode = "400", description = "Anfrageparameter sind fehlerhaft",
+                            content = @Content(mediaType = "application/json", schema = @Schema(implementation = WlsExceptionDTO.class))
+                    ),
+                    @ApiResponse(
+                            responseCode = "500", description = "Probleme bei der Kommunikation mit dem externen System von dem die Daten importiert werden",
+                            content = { @Content(mediaType = "application/json", schema = @Schema(implementation = WlsExceptionDTO.class)) }
                     )
             }
     )
-    @GetMapping("/ereignisse/{wahlbezirkID}")
+    @GetMapping("/{wahlbezirkID}")
     public ResponseEntity<WahlbezirkEreignisseDTO> getEreignis(@PathVariable("wahlbezirkID") String wahlbezirkID) {
         val ereignisFromService = ereignisService.getEreignis(wahlbezirkID);
-        return withBodyOrNoContent(ereignisFromService.map(ereignisDTOMapper::toDTO));
+        return returnsOkWithBodyOrNoContent(ereignisFromService.map(ereignisDTOMapper::toDTO));
     }
 
-    /**
-     * This BusinessAction's purpose is: Speichern von besonderen Ereignissen
-     */
     @Operation(description = "Speichern der Ereignisse des Wahllokals {wahlbezirkID}")
     @ApiResponses(
             value = {
                     @ApiResponse(
                             responseCode = "200", description = "OK",
                             content = { @Content(mediaType = "application/json", schema = @Schema(implementation = WahlbezirkEreignisseDTO.class)) }
                     ),
+                    @ApiResponse(
+                            responseCode = "400", description = "Anfrageparameter sind fehlerhaft",
+                            content = @Content(mediaType = "application/json", schema = @Schema(implementation = WlsExceptionDTO.class))
+                    ),
                     @ApiResponse(
                             responseCode = "500", description = "Probleme bei der Kommunikation mit dem externen System von dem die Daten importiert werden",
                             content = { @Content(mediaType = "application/json", schema = @Schema(implementation = WlsExceptionDTO.class)) }
                     )
```

Comment:
Der Service hat keine Abhängigkeit zu anderen Services. Ein 500er kann daher nur bei sonstigen internen Problemen auftreten.

Reply:
siehe commit [992d6ba](https://github.com/it-at-m/Wahllokalsystem/pull/412/commits/992d6ba909338cab856f1c85081656e3844bbaa8)